book/forms.rst

```diff
@@ -1048,7 +1051,8 @@ that will house the logic for building the task form::
             $builder
                 ->add('task')
                 ->add('dueDate', null, array('widget' => 'single_text'))
-                ->add('save', 'submit');
+                ->add('save', 'submit')
```

Comment:
Jumped semicolon to a next line as in this case it might not be the last call.
Also it might be useful to remove submit methods as by best practices they should be included in the template